specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/searchindex.json

```diff
@@ -339,16 +397,204 @@
         }
       }
     },
+    "/docs/suggest": {
+      "get": {
+        "tags": [
+          "Documents"
+        ],
+        "operationId": "DocumentsProxy_SuggestGet",
+        "externalDocs": {
+          "url": "https://docs.microsoft.com/rest/api/searchservice/suggestions"
+        },        
+        "x-ms-examples": {
+          "SuggestIndexGetSearch": { "$ref": "./examples/SearchIndexGetSuggest.json" }
+        },
+        "description": "Suggests query terms based on input text and matching documents in the Azure Search index.",
+        "parameters": [
+          {
+            "name": "search",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The search text to use to suggest queries. Must be at least 1 character, and no more than 100 characters.",
+            "x-ms-client-name": "searchText"
+          },
+          {
+            "name": "suggesterName",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The name of the suggester as specified in the suggesters collection that's part of the index definition."
+          },
+          {
+            "name":"$filter",
+            "in": "query",
+            "type": "string",
+            "description": "An expression that filters the documents considered for suggestions.",
+            "x-ms-client-name": "Filter",
+            "x-ms-parameter-grouping": {
+              "name": "SuggestParameters"
+            }
+          },
+          {
+            "name":"fuzzy",
+            "in": "query",
+            "type": "boolean",
+            "description": "A value indicating whether to use fuzzy matching for the suggestions query. Default is false. When set to true, the query will find terms even if there's a substituted or missing character in the search text. While this provides a better experience in some scenarios it comes at a performance cost as fuzzy autocomplete queries are slower and consume more resources.",
+            "x-ms-client-name": "UseFuzzyMatching",
+            "x-ms-parameter-grouping": {
+              "name": "SuggestParameters"
+            }
+          },
+          {
+            "name":"highlightPostTag",
+            "in": "query",
+            "type": "string",
+            "description": "A string tag that is appended to hit highlights. Must be set with HighlightPreTag. Default is \"\".",
```

Comment:
Let's say "Default is the empty string" instead, since it's more programming-language-neutral. Same for `highlightPreTag`.

Reply:
Good suggestions. The doc on the POST payload says "If omitted, hit highlighting of suggestions is disabled." I like this one better. What do you think?

Reply:
Agreed, that's more descriptive. Let's go with that.